### PR TITLE
Make intro more accessible

### PR DESCRIPTION
**Pull request by martinthomson**

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -128,15 +128,16 @@ general-purpose secure transport for multiple applications.
...
-transport protocols.  QUIC uses UDP as substrate so as to not require changes to
-legacy client operating systems and middleboxes to be deployable.  QUIC
-authenticates all of its headers and encrypts most of the data it exchanges,
-including its signaling.  This allows the protocol to evolve without incurring a
-dependency on upgrades to middleboxes.  This document describes the core QUIC
-protocol, including the conceptual design, wire format, and mechanisms of the
-QUIC protocol for connection establishment, stream multiplexing, stream and
-connection-level flow control, connection migration, and data reliability.
+QUIC uses UDP as substrate so as to not require changes to legacy client
+operating systems and middleboxes to be deployable.  QUIC authenticates all of
```
Comment (mikkelfj) [inline draft-ietf-quic-transport.md]: that sensent is hard to parse

Comment (ianswett) [reply draft-ietf-quic-transport.md]: Agreed, this sentence is awkward.

How about "QUIC uses UDP as a substrate to avoid changes to legacy client operating systems and middleboxes in order to be deployed."

**Comment by hardie** [comment]
On Mon, Sep 24, 2018 at 3:51 PM, ianswett <notifications@github.com> wrote:

> *@ianswett* commented on this pull request.
> ------------------------------
>
> In draft-ietf-quic-transport.md
> <https://github.com/quicwg/base-drafts/pull/1789#discussion_r220013359>:
> Agreed, this sentence is awkward.
>
> How about "QUIC uses UDP as a substrate to avoid changes to legacy client
> operating systems and middleboxes in order to be deployed."
>
>
How about just "QUIC uses UDP as a substrate to avoid requiring changes in
legacy client operating systems or middleboxes."?  I think the point about
deployment is made by the "avoid requiring changes", so I think it could be
dropped.

Ted
